finish/getting-started-with-bsr/go.mod

```diff
@@ -1,14 +0,0 @@
-module github.com/bufbuild/buf-tour
```

Comment:
I thought that having `<USER>` would be confusing, since it won't match the user's actual entry and we can't pull it in here from the BSR like we can on the website. Have reconsidered and changed it to `<USERNAME>` to at least match our other examples.